examples/with-tailwind/apps/docs/next.config.ts

```diff
@@ -2,4 +2,10 @@
 module.exports = {
```

Comment:
This is renamed to `.ts` but still using CJS. It should `export default {...}` instead! 

We can also remove the type annotation and use `import type { NextConfig } from 'next';` instead.